janitor.go

```diff
@@ -104,7 +104,7 @@ func (j *Janitor) handleVM(ctx context.Context,
 
 	if bootTime != nil {
 		bootedAgo := time.Now().UTC().Sub(*bootTime)
-		logger.WithField("booted_ago", bootedAgo).Info("instance booted ago")
+		logger.WithField("booted_ago", bootedAgo).Info("instance boot time")
```

Comment:
Not sure if "boot time" is the best word for it, since it shows the duration since the instance was booted ("boot time" to me sounds like "time it took for the instance to boot").